### DISCUSSION TITLE
Planner with no schedule

### DISCUSSION THREAD

**Discussion by khaledar-personal** [kudrykv/latex-yearly-planner#89] [Ideas]
Hi, I absolutely love this planner, thank you so much for all the hard work.

I'm wondering if there's a way to have a version that essentially has:
1. No schedule portion on the daily pages
2. Top priorities segment is maybe 15 items instead of the current 8.
3. More notes per day (i believe there's already a version that has this, but without the above 2)

I usually have my schedule on Google calendar, but I use the planner for TODO's and taking notes & daily journaling.

If this is difficult and there's a way for me to do it or to help creating please let me know

Much appreciated

**Comment by kudrykv** [comment]
Hi!

For more notes per day, you need to edit this piece of code
https://github.com/kudrykv/latex-yearly-planner/discussions/80#discussioncomment-5939517

Daily page starts with this tex/go template https://github.com/kudrykv/latex-yearly-planner/blob/a321d94fc4edf271ac564b232d356978b91578ed/tpls/_common_05_daily.tpl
Here, you can remove the schedule
